Collapsible/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React, {Component} from 'react'
+import {Motion, spring} from 'react-motion'
+
+export default class Collapsible extends Component {
+  constructor () {
+    super()
+
+    this.state = {
+      height: 0
+    }
+  }
+
+  componentDidMount () {
+    this.setState({
+      height: getHeight(this.content)
+    })
+
+    window.addEventListener('resize', () => {
+      const height = getHeight(this.content)
+
+      if (this.state.height !== height) {
+        this.setState({ height })
```

Comment:
This is going to cause the `render` method to get called on every resize event, which you might have happen more than once per actual browser rendering. Since `getBoundingClientRect` causes a layout, this is very expensive. Check #144 to see how I debounced the event so it only happens when the browser will actually render a new frame. It made a world of difference in some browsers.